pyproject.toml

```diff
@@ -115,3 +122,16 @@ known_vellum = ["vellum", "vellum_cli", "vellum_ee"]
 default_section = "THIRDPARTY"
 sections = ["FUTURE", "TESTING", "STDLIB", "TYPING", "THIRDPARTY", "VELLUM", "FIRSTPARTY", "LOCALFOLDER"]
 no_lines_before = ["STDLIB", "TYPING"]
+
+[tool.black]
+line-length = 120
+target-version = [
+    "py38",
+]
+include = ".*\\.py$"
+exclude = "(.git|venv|src/vellum/client|tests/client|scripts)"
```

Comment:
should we auto format/auto import sort the fern generated files in our post processing script so that we could remove these exclusions?